root/static/scripts/release-editor/fields.js

```diff
@@ -296,6 +297,10 @@ class Track {
         return hasVariousArtists(this.artistCredit());
     }
 
+    hasFeatOnTitle() {
+        return featRegex.test(this.name().replace(/[\[［\]］(（)）]/g,' '));
+    }
+
```

Comment:
Did not test but that might be a way to use `bracketPairs`:

```suggestion
    const bracketRegex = new RegExp(bracketPairs.flat().join(''), 'g');

    hasFeatOnTitle() {
        return featRegex.test(bracketRegex,' '));
    }

```

Reply:
They still have to be in a character class and escaped with `escapeRegExp` if you wanna do that. (Also, the brackets are meant to be removed from the name. :))

Reply:
Do you feel it's worth the extra effort as opposed to just listing them again there?

Reply:
Not really, but a comment saying to keep them in sync could be helpful.

Reply:
Added comments

Reply:
> Do you feel it's worth the extra effort

The suggested code might need to be updated again, but I did update it immediately after the next comment, so the “extra effort” should be hopefully limited.

> Added comments

AFAICT, in the current commit https://github.com/metabrainz/musicbrainz-server/pull/1546/commits/71706724264a2ec195a6528b083e19f07f8b159b, comments have been added to the various `bracketPairs`, not to `hasFeatOnTitle`.

Reply:
I was confused because it didn't seem to be working, but it just needed the result of `escapeRegExp` to be wrapped in `[]`. Thanks!